R/gee-tidiers.R

```diff
@@ -0,0 +1,135 @@
+#' @templateVar class gee
+#' @template title_desc_tidy
+#'
+#' @param x A `gee` object returned from a call to [gee::gee()].
+#' @param conf.int 	Logical indicating whether or not to include a confidence
```

Comment:
Would you use templates for parameter documentation for the standard arguments? See https://github.com/tidymodels/broom/blob/master/R/biglm-tidiers.R#L6 for an example